### PR TITLE
BZ-1666563: Added steps for adding registry's CA to verify image sign…

### PR DESCRIPTION
**Pull request by bergerhoffer**
…ature

https://bugzilla.redhat.com/show_bug.cgi?id=1666563

Preview: http://file.rdu.redhat.com/~ahoffer/2020/image_signatures.html#verifying-image-signatures-using-openshift-cli

### CODE REVIEW

**Comment by bergerhoffer** [comment]
Hi @dmage - would you be able to review this doc update dealing with verifying image signatures? It incorporates the steps from this KCS [1].

[1] https://access.redhat.com/solutions/3809961

**Comment by bergerhoffer** [comment]
@wzheng1 could you also please review this from a QE perspective? Thanks!

Preview: http://file.rdu.redhat.com/~ahoffer/2020/image_signatures.html#verifying-image-signatures-using-openshift-cli

**Comment by lamek** [comment]
LGTM

File: admin_guide/image_signatures.adoc [hunk 1]
```diff
@@ -152,6 +152,24 @@ $ oc adm verify-image-signature \
...
+[NOTE]
+====
+If the `oc adm verify-image-signature` command returns an `x509: certificate signed by unknown authority` error, you might need to add the registry's certificate authority (CA) to the list of CAs trusted on the system. You can do this by performing the following steps:
+
+. Copy the CA certificate to the *_/etc/pki/ca-trust/source/anchors/_* directory. For example:
++
+----
+# cp /etc/docker/certs.d/docker-registry.default.svc\:5000/node-client-ca.crt \
+    /etc/pki/ca-trust/source/anchors/
+----
```
Comment (wzheng1) [inline admin_guide/image_signatures.adoc:164]: This node-client-ca.crt is from cluster, user need to copy the crt file to the system where oc is running on, maybe need scp.

**Comment by bergerhoffer** [comment]
@wzheng1 @dmage I've updated to add a step to transfer the file. Can you please see if this looks good now? Thanks!

**Comment by bergerhoffer** [comment]
Sorry, probably helps to actually push the changes... :). PR is updated now @wzheng1, @dmage .

**Comment by dmage** [comment]
Looks good to merge

**Pull request merged**